components/tidb_query/src/codec/data_type/vector.rs

```diff
@@ -351,6 +351,38 @@ impl VectorValue {
             }
         }
     }
+
+    pub fn encode_sort_key(
```

Comment:
The name might need to to be changed. this one not only encode the sort key, but only encode other kind of datums.

Reply:
I think all kinds of data can be considered as sort key, but `Bytes`, as a special case, is different from the original data.